liquibase-standard/src/main/java/liquibase/Liquibase.java

```diff
@@ -1199,26 +1196,16 @@ public void reportUnexpectedChangeSets(boolean verbose, Contexts contexts, Label
     }
 
     /**
-     * Sets checksums to null so they will be repopulated next run
+     * Sets checksums to null, so they will be repopulated next run
+     *
+     * @deprecated Use {@link CommandScope(String)}
      */
+    @Deprecated
     public void clearCheckSums() throws LiquibaseException {
-        LOG.info("Clearing database change log checksums");
-        runInScope(() -> {
-            LockService lockService = LockServiceFactory.getInstance().getLockService(database);
-            lockService.waitForLock();
-
-            try {
-                checkLiquibaseTables(false, null, new Contexts(), new LabelExpression());
-                Scope.getCurrentScope().getSingleton(ChangeLogHistoryServiceFactory.class).getChangeLogService(database).clearAllCheckSums();
-            } finally {
-                try {
-                    lockService.releaseLock();
-                } catch (LockException e) {
-                    LOG.severe(MSG_COULD_NOT_RELEASE_LOCK, e);
-                }
-            }
-            resetServices();
-        });
+        CommandResults commandResults = new CommandScope("clearChecksums")
```

Comment:
Though not critical, it would be ideal to reference the `COMMAND_NAME` variable from `ClearChecksumsCommandStep.java`.

Reply:
Yeap, it's better to use the constant here to prevent future problems.

Reply:
Done, thank you guys!